Jumpscale/clients/blockchain/tfchain/README.md

```diff
@@ -684,3 +684,40 @@ unconfirmed  Tx: 573290763024ae0a5e981412598a3d41bc02f8da628fa1e1adfe07d98818c68
         > to: 0125c0156f6c1c0bc43c7d38e17f8948300564bef63caac05c08b0fd68996e494704bbbe0268cb
         > from: 01773a1dd123347e1030f0822cb8d22082fe3f9b0ea8563d4ac8e7abc377eba920c47efb2fd736
 ```
+
+### Capacity reservation
+
+During the fist beta phase of the public launch of the TF grid, beta tester will be able to reserve 2 kind of workload on the grid.
+
+- Zero-OS virtual machines
+- S3 archive storage instances
+
+**At the time of writing, everything happens on the testnet network**
+**Don't send real TFT from the main network !!**
+
+#### How to reserve some capacity on the Threefold Grid
+
+To be able to make a reservation you first need to:
+
+- have a wallet with some fund
+- record a threebot on the tfchain
+
+Once you have both, you can then use your wallet client to do a reservation.
+The wallet expose a module called `capacity`. On this module you will find function to reserver the different type of capacity.
+
+Examples:
+
+```python
+c = j.clients.tfchain.myclient
+w = c.wallets.mywallet
+tx_id = w.capacity.reserve_s3(
+    size=1, # each workload have a different size available
+    email='user@email.com', # the email on which you will received the connection information
+    bot_id='my3bot.example.org') # your threebot id, it can be any of the names you gave to your 3bot
+```
+
+
+The result of the `reserve_s3` method call is the transaction ID in which you reservation has been created.
```

Comment:
no need to use the web explorer though. You can also do it using our JSX client:

```python
j.clients.tfchain.myclient.transaction_get(txid)
```

Reply:
good I'll add that too